components/popconfirm/nz-popconfirm.directive.ts

```diff
@@ -39,7 +39,7 @@ import { NzPopconfirmComponent } from './nz-popconfirm.component';
 export class NzPopconfirmDirective extends NzTooltipBaseDirective implements OnInit {
   @Input('nzPopconfirmTitle') specificTitle: NzTSType;
   @Input('nz-popconfirm') directiveNameTitle: NzTSType | null;
-  @Input('nzPopconfirmTrigger') specificTrigger: NzTooltipTrigger;
+  @Input('nzPopconfirmTrigger') specificTrigger: NzTooltipTrigger | null;
```

Comment:
You don't have to add `null` here since `NzTooltipTrigger` is defined as:

```ts
export type NzTooltipTrigger = 'click' | 'focus' | 'hover' | null;
```